src/App.php

```diff
@@ -128,8 +125,8 @@ private function getEventManager()
      */
     public function run($request = null, $response = null)
     {
-        ($request != null) ?: $request = $this->request;
-        ($response != null) ?: $response = $this->response;
+        ($request != null) ?: $request = ServerRequestFactory::fromGlobals();
+        ($response != null) ?: $response = new Response();
```

Comment:
I think it can be more readable with:
```php
$request = ($request !== null) ?: ServerRequestFactory::fromGlobals();
$response = ($response !== null) ?: new Response();
```
also, `!==` instead of `!=`